lighthouse-core/report/templates/report-template.html

```diff
@@ -61,15 +61,19 @@
             <h1 class="menu__header-title">Lighthouse</h1>
             <div class="menu__header-version">Version: {{lighthouseVersion}}</div>
           </div>
-          <ul class="menu__nav">
-            {{#each aggregations}}
-            <li class="menu__nav-item">
-              <a class="menu__link" href="#{{nameToLink this.name}}">
-                {{ this.name }}
-              </a>
-            </li>
-            {{/each}}
-          </ul>
+          {{#if_eq reportContext "perf-x" }}
```

Comment:
Based on the loop refactor it seems like the left-nav partial should be pretty straightforward. I like tossing it into the `report-template.html` directly. 

And yeah, report-generator can take the otherReports array now and handle that.  Seems good!